### DISCUSSION TITLE
hstore Postgres workaround?

### DISCUSSION THREAD

**Discussion by davidboom95** [prisma/prisma#2863] [Q&A] [answered]
Hi, There is any way of storing hstore values using prisma? I've tried:
- Changing column schema to `String`
- Saving the column value parsed as a hstore string

None of this works, it gives me this error

```
  Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: QueryError(Error { kind: Db, cause: Some(DbError { severity: "ERROR", parsed_severity: Some(Error), code: SqlState("54000"), message: "number of pairs (577137010) exceeds the maximum allowed (26843545)", detail: None, hint: None, position: None, where_: None, schema: None, table: None, column: None, datatype: None, constraint: None, file: Some("hstore_io.c"), line: Some(448), routine: Some("hstore_recv") }) }) })
    at PrismaClientFetcher.request (/var/task/node_modules/@prisma/client/src/runtime/getPrismaClient.ts:766:15)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
```

Maybe as a raw query works?

**Comment by ryands17** [comment, marked as answer]
Hey @davidboom95 :wave: 
Yes as a workaround, you could try `prisma.queryRaw`, but could you also open a feature request [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) regaring this use case?

**Reply by ryands17** [reply]
@davidboom95 did you get it work with any workaround?

**Reply by rymnc** [reply]
Is there any workaround found yet?

**Comment by ludralph** [comment]
Hi @davidboom95,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
